lib/report.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const _ = require('lodash');
+const path = require('path');
+
+const ONLINE_WEB = path.join(process.env.REI_CODE_HOME, 'online', 'online-web');
```

Comment:
does this end up being the same as the package root ../ ? might be able to make this more generic without the path hard coding here

Reply:
I'll remove this actually, I was trying to display the relative path from `/online-web` but this still doesn't display that when I'm running from another directory. Keep it simple and just display the entire path to files.